_storage/gaia-admin.md

```diff
@@ -0,0 +1,338 @@
+  "apiKeys": [ "hello" ],
+  "gaiaSettings": {
+    "configPath": "/tmp/hub-config/config.json"
+  },
+  "reloadSettings": {
+    "command": "/bin/sh",
+    "argv": [
+      "-c",
+      "docker restart docker_hub_1 &"
+    ],
+    "env": {},
+    "setuid": 1000,
+    "setgid": 1000
+  }
+}
+```
+
+The `port` is the port where Gaia is running. The `apiKeys` field is key used for making calls to the hub.  The `gaiaSettings`
+
+
+The `argsTransport` section configures the hub logging. The service uses the `winston` logging service. Refer to their documentation for full details on the [logging configuration options](https://github.com/winstonjs/winston).
+
+<table class="uk-table uk-table-small uk-table-divider">
+   <thead>
+      <tr>
+         <th>Field</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td><code>level</code></td>
+         <td>Lowest level this transport will log. (default: <code>info</code>)</td>
+      </tr>
+      <tr>
+         <td><code>handleException</code></td>
+         <td>Set to true to have this transport handle exceptions. (default: <code>false</code>)</td>
+      </tr>
+      <tr>
+         <td><code>timestamp</code></td>
+         <td>The timestamp when the message was received.</td>
+      </tr>
+      <tr>
+         <td><code>stringify</code></td>
+         <td>Converts the output to a JSON string.</td>
+      </tr>
+      <tr>
+         <td><code>colorize</code></td>
+         <td>Colorizes the standard logging level</td>
+      </tr>
+      <tr>
+         <td><code>json</code></td>
+         <td>Log format.</td>
+      </tr>
+   </tbody>
+</table>
+
+The `reloadSettings` configure the command that is used to reload your Gaia hub.
+
+<table class="uk-table uk-table-small uk-table-divider">
+   <thead>
+      <tr>
+         <th>Field</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td><code>command</code></td>
+         <td>A command which reloads the Gaia hub service.</td>
+      </tr>
+      <tr>
+         <td><code>argv</code></td>
+         <td>An array containing the command arguments.</td>
+      </tr>
+      <tr>
+         <td><code>env</code></td>
+         <td>This is a key/value list of any environment variables
+           that need to be set for the command to run. This is optional.</td>
+      </tr>
+      <tr>
+         <td><code>setuid</code></td>
+         <td>This is the UID under which the command will be run. This is optional.</td>
+      </tr>
+      <tr>
+         <td><code>setgid</code></td>
+         <td>This is the GID under which the command will run. This is optional.</td>
+      </tr>
+   </tbody>
+</table>
+
+
+## Using the admin service APIs
+
+You use the admin service APIs to manage the hub. Get calls require that you pass the `API_KEY` set in the admin service config:
+
+```bash
+$ export API_KEY="hello"
+```
+
+`POST` calls require that you supply your AWS access and secret key pair.
```

Comment:
We don't need to set the AWS access and secret key pairs for these CURL requests.

Reply:
@kantai is that true even if I am making the call say from my local machine as opposed to from the instance terminal?

Reply:
Yep -- the AWS access and secret key pairs are only used if you are changing the driver backend to "aws" from "disk" (the change backend example later on in the doc uses those variables), so they should only be discussed in that section.

Reply:
removed those.